src/plugins/timelist/inspector/TimelistPropertiesView.vue

```diff
@@ -30,6 +30,15 @@
             These settings don't affect the view while editing, but will be applied after editing is
```

Comment:
I believe the intention here is not to apply the change till it is saved. I don't think this was done because the change couldn't take effect immediately.

Reply:
Why though?

Reply:
Because it is jarring to the user. @charlesh88 Can comment to this. If he's OK changing this behavior I have no issues making it consistent.

Reply:
Up to @charlesh88. [He's the one who added it in](https://github.com/nasa/openmct/commit/a99a6b3c828d1de38087c8218007c8445e8a501e) :)

I'll break this out as a separate issue since it's unrelated to this PR, though.